cmd/regctl/cmd/apply/testdata/manifest.yaml

```diff
@@ -12,7 +12,7 @@ data:
       dependencies:
         - pattern: $resource.spec
           filter: mime_type.contains('openapi')
-      action: registry compute lint $resource.spec --linter spectral
```

Comment:
@shrutiparabgoogle renaming `registry` forces us to update the manifest actions, which might be hard to change. What would you think about making the controller resilient to this by having it automatically replace "registry" with "regctl" (when it is at the start of an action string)?

Reply:
Right - I agree that we should fix the manifests also.

Reply:
Addressed here: https://github.com/apigee/registry/pull/571/commits/b52caed59a1d8a34268683bebdd2e8f71a8581c9